recipes/sorl-thumbnail/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "sorl-thumbnail" %}
+{% set version = "12.6.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 66771521f3c0ed771e1ce8e1aaf1639ebff18f7f5a40cfd3083da8f0fe6c7c99
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=2.7
+    - pip
+  run:
+    - django >=1.11
+    - python >=2.7
+    - pillow
```

according to the package dependency, they just need ``python`` at runtime, or did I miss something?
```suggestion
    - python >=3.4
```

---

it seems that ``django`` and ``pillow`` are optional

---

Yes they are optional for the package to be installed, but they are recommended for the package to be used.

---

It is up to you to add them as a maintainer them
If you think it is better with it, please go ahead and readd them :)
Just pillow I think would be nice to remove them, because it seems this package supports several libraries to manipulate images

---

okay thanks a lot